.librarian/puppet/config

```diff
@@ -1,4 +1,4 @@
 ---
-LIBRARIAN_PUPPET_DESTRUCTIVE: 'false'
+LIBRARIAN_PUPPET_DESTRUCTIVE: 'true'
```

Comment:
I don't think this should have changed. govuk_puppet has "NB: There should *never* be any changes to `.librarian/puppet/config`." for example.